Auth0/ObjectiveC/A0RSA.m

```diff
@@ -38,6 +39,17 @@ - (nullable instancetype)initWithKey: (SecKeyRef)key {
 }
 
 - (NSData *)sign:(NSData *)plainData {
+    #if TARGET_OS_OSX
+    NSData * signature;
+    CFErrorRef error = NULL;
+
+    signature = CFBridgingRelease(SecKeyCreateSignature(self.key,
+                                                        kSecKeyAlgorithmRSASignatureDigestPKCS1v15SHA256,
+                                                        (__bridge CFDataRef) plainData,
+                                                        &error));
+
+    return signature;
+    #else
```

Comment:
`SecKeyRawSign ` was never officially supported on macOS and on the latest versions it was removed. So here we add an implementation with a mac0S-supported API.

Reply:
Could argue this should be a separate PR dealing with this migration for macOS support in Xcode 12. Please mention in changes.

Reply:
I added it to the changes section. This was not changed arbitrarily, but because otherwise the library would not compile for macOS and I wouldn't be able to test it with Xcode 12.